tests/unit/services/unified-event-handler-test.js

```diff
@@ -94,7 +99,7 @@ test('register binds multiple callbacks to the event on the specified target but
 
   let callback1Stub = sandbox.stub();
   let callback2Stub = sandbox.stub();
-  let triggerSpy = sandbox.spy(service, 'triggerEvent');
+  let triggerSpy = sandbox.spy(service, '_runThrottler');
```

Comment:
I'm not sure spying on this does anything more then couple the tests to the internal implementation. 

As long as the callbacks are invoked appropriately the test is doing its job, any extra stuff is merely tech debt.

Reply:
Yeah, I agree. No point in spying on an internal method, esp. since there is essentially no logic between the tested method and the spied method.